Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -1551,9 +1551,6 @@
          * numbers for those locations should be 0.
          */
         #ifdef VALIDATE_MESH_TILT
-          #if ENABLED(Z_SAFE_HOMING)
-            constexpr xy_float_t safe_homing_xy = { Z_SAFE_HOMING_X_POINT , Z_SAFE_HOMING_Y_POINT };
-          #endif
```

Comment:
Its actually no longer used anywhere except the G29 debug output, which in reality should be modified as well to take the variables into account when home offsets exist.

Reply:
The `Z_SAFE_HOMING_[XY]_POINT ` values are defined in terms of native machine space. The `safe_homing_xy` global `constexpr` defined in `motion.h` is used in `home_z_safely` and also by UBL with `VALIDATE_MESH_TILT`.